distribution/docs/src/main/resources/content/_integrating/_endpoints/catalog-rest-endpoint.adoc

```diff
@@ -3,291 +3,320 @@
 
-.Request with Missing XML Prologue
-If a request with a missing XML prologue is sent to the Catalog ReST endpoint,
-the metacard is created successfully.
+.Retrieve Resource Request URL
+[source,http]
+----
+https://<FQDN>:<PORT>/services/catalog/<metacardId>?transform=resource
+----
 
-.Request with Non-XML Data
-If a request with non-XML data sent to the Catalog ReST endpoint, the metacard will be created and the content will stored in the `metadata` field.
-
-'''
-
-//https://github.com/codice/ddf/issues/4374
-//.[[_catalog_rest_update_example]]Update Operation Examples
-//notes
-//
-//.Update Request Example
-//[source,javascript,linenums]
-//----
-//tbd
-//----
-//
-//.Update Response Example
-//[source,javascript,linenums]
-//----
-//tbd
-//----
-//'''
-//.[[_catalog_rest_delete_example]]Delete Operation Examples
-//notes
-//
-//.Delete Request Example
-//[source,javascript,linenums]
-//----
-//tbd
-//----
-//
-//.Delete Response Example
-//[source,javascript,linenums]
-//----
-//tbd
-//----
+See <<{architecture-prefix}available_metacard_transformers,Metacard Transformers>> for details on metacard transformers.
 
-.[[_catalog_rest_read_example]]Read Operation Examples
+.Read Error Response Examples
+If the metacard is not returned successfully, check for these errors.
 
-The `read` operation can be used to retrieve metadata in different formats.
+.Create Error Response Examples
+[cols="2,2,3" options="header"]
+|===
+|Status Code
+|Error Message
+|Possible Causes
+
+|`404 Not Found`
+|`<pre>Unable to retrieve requested metacard.</pre>`
+|Invalid Metacard ID
+
+.3+.^|`500 Server Error`
+|`<pre>Unknown error occurred while processing request.</pre>`
+|Transformer is invalid, unsupported, or not configured.
+|`<pre>Unable to transform Metacard.  Try different transformer: </pre>`
+|Metacard does not have an associated resource (is metadata only).
+|`<pre>READ failed due to unexpected exception: </pre>`
+|Invalid source ID, or source unavailable.
 
-. Make a read request to the REST URL specifying the catalog id.
-. Add a transform query parameter to the end of the URL specifying the shortname of the transformer to be used (e.g., `transform=kml`).
+|===
 
-[NOTE]
-====
-Not all installations will have all transformers installed. Contact ${branding} administrator for available transformers.
-====
+.[[_catalog_rest_update_example]]Catalog Rest Update Operation Examples
+To update the metadata for a metacard, send a `PUT` request with the ID of the Metacard to be updated appended to the end of the URL
+and the updated metadata is contained in the HTTP body.
 
-.Read Request Example
-[source,javascript,linenums]
+Optionally, specify the transformer to use when parsing an override of a metadata attribute.
+
+.Update Request URL
+[source,http]
 ----
-${public_url}/services/catalog/<metacardId>
+{url}<metacardId>?transform=<input transformer>
```

Comment:
❓ Are we going to update the content of some of these things? It's confusing that some of the example request URLs have `https://<FQDN>:<PORT>/services/catalog` as the base and then this one and others just have `{url}`. I'd lean towards the former, but either way they should at least be consistent.

Reply:
agreed. I'll go with the `<>` syntax because that's what we use elsewhere`

Reply:
I see what this comment meant. The `{url}` usage just resolves to the full `:url:` property listed above. I can change it if it helps maintainability. (We do have other places where we used the syntax `{FQDN}` and I was thought that's what you were referring to and what I'll be standardizing.

Reply:
oh, there are places where the property does not resolve. I don't use it in the templates anyway, so I've dropped it and just spelled it out where needed.